x/validators/model.go

```diff
@@ -69,13 +70,13 @@ func GetAccounts(bucket orm.Bucket, kv weave.KVStore) (*Accounts, error) {
 	}
 
 	if res == nil {
-		return nil, ErrNotFound("account")
+		return nil, errors.NotFoundErr.New("account not found")
 	}
 	switch t := res.Value().(type) {
 	case *Accounts:
 		return t, nil
 	default:
-		return nil, ErrWrongType(t)
+		return nil, errors.WithType(errors.InvalidTypeErr, t)
```

Comment:
I like this generic function WithType

Reply:
But... if we are usually wrapping `errors.InvalidTypeErr`, why not a helper in errors for just this:

`errors.InvalidType(t)` to do the same thing... and you can use this longer form if you have a different base class of error to use

Reply:
@ethanfrey I really don't like introducing a lot of helpers that are tied to a specific error, because it gives us back what we had before. 
I'd rather use a bit more generic functions for transparency, rather than having a bunch of helpers, it could be misleading and hard to maintain.

Thoughts?
BTW I'm merging this into my x PR to refactor errors.

Reply:
I'll keep it as is for now, I have fixed other comments in this PR and merging it into x for further refactoring

Reply:
Sure, I get your point, and if we make it default one way, people get lazy and don't set the proper base error.

Anyway, it was more a comment for possible helper, I like the functionality as is